packages/ember-runtime/lib/ext/rsvp.js

```diff
@@ -54,32 +31,39 @@ export function onerrorDefault(reason) {
     return;
   }
 
+  // seems like we should turn this into an error too?
   if (error && error.name !== 'TransitionAborted') {
-    if (Ember.testing) {
-      // ES6TODO: remove when possible
-      if (!Test && has(testModuleName)) {
-        Test = require(testModuleName)['default'];
-      }
-
-      if (Test && Test.adapter) {
-        Test.adapter.exception(error);
-        Logger.error(error.stack);
-      } else {
-        throw error;
-      }
-    } else if (Ember.onerror) {
-      Ember.onerror(error);
-    } else {
-      Logger.error(error.stack);
-    }
+    dispatchError(error);
   }
+  // let onerror = getOnerror();
+  //  if (error && error.name !== 'TransitionAborted') {
+  //   if (isTesting()) {
+  //     // ES6TODO: remove when possible
+  //     if (!Test && has(testModuleName)) {
+  //       Test = require(testModuleName)['default'];
+  //     }
+
+  //     if (Test && Test.adapter) {
+  //       Test.adapter.exception(error);
+  //       Logger.error(error.stack);
+  //     } else {
+  //       throw error;
+  //     }
+  //   } else if (onerror) {
+  //     onerror(error);
+  //   } else {
+  //     Logger.error(error.stack);
+  //   }
+  // }
 }
 
-export function after (cb) {
-  run.schedule(run.queues[run.queues.length - 1], cb);
+export function after(cb) {
+  run.backburner.schedule('rsvpAfter', null, cb);
 }
 
 RSVP.on('error', onerrorDefault);
 RSVP.configure('after', after);
 
+run._addQueue('rsvpAfter', 'destroy');
```

Comment:
Can we use one of our fake symbols here instead of `rsvpAfter`?  I would hate to have people hook into this themselves, we should make it super clear that it is private...